Apps/fastlane/Fastfile

```diff
@@ -47,7 +47,7 @@ lane :build do
   )
 
   UI.important("Updating the SDK's source code version to non-production version. This allows the sample apps to show the SDK version at runtime for app user to better understand the version of the SDK they are running.")
-  sh("../../../scripts/update-version.sh \"#{new_app_version} (#{new_build_number})\"")
+  sh("../../../scripts/update-version.sh \"#{new_app_version} (#{new_build_number})\" || true") # we add "|| true" to the end to prevent the script from throwing any errors. SDK wrappers use this Fastfile and they might not have this script in the project. 
```

Comment:
I believe this is done not to halt the process. if so why not just add a check instead of this? instead of `|| true` which gives it a blanket error pass 

```
version_update_script_path = '../../../scripts/update-version.sh'
if File.exist?(version_update_script_path)
  sh("#{version_update_script_path} \"#{new_app_version} (#{new_build_number})\"")
```

Reply:
I like that idea. 

I implemented the change in latest commit.